doc/dependencies.md

```diff
@@ -15,35 +15,35 @@ You can find installation instructions in the `build-*.md` file for your platfor
 
 ## Required
 
-| Dependency | Version used | Minimum required | Runtime |
-| --- | --- | --- | --- |
-| [Boost](https://www.boost.org/users/download/) | 1.77.0 | [1.64.0](https://github.com/bitcoin/bitcoin/pull/22320) | No |
-| [libevent](https://github.com/libevent/libevent/releases) | 2.1.12-stable | [2.1.8](https://github.com/bitcoin/bitcoin/pull/24681) | No |
-| [glibc](https://www.gnu.org/software/libc/) | N/A | [2.18](https://github.com/bitcoin/bitcoin/pull/23511) | Yes |
+| Dependency | Releases | Version used | Minimum required | Runtime |
+| --- | --- | --- | --- | --- |
+| [Boost](../depends/packages/boost.mk) | [link](https://www.boost.org/users/download/) | [1.77.0](https://github.com/bitcoin/bitcoin/pull/24383) | [1.64.0](https://github.com/bitcoin/bitcoin/pull/22320) | No |
+| [libevent](../depends/packages/libevent.mk) | [link](https://github.com/libevent/libevent/releases) | [2.1.12-stable](https://github.com/bitcoin/bitcoin/pull/21991) | [2.1.8](https://github.com/bitcoin/bitcoin/pull/24681) | No |
+| glibc | [link](https://www.gnu.org/software/libc/) | [2.18](https://github.com/bitcoin/bitcoin/pull/23511) | Yes |
```

Comment:
```suggestion
| glibc | [link](https://www.gnu.org/software/libc/) | N/A | [2.18](https://github.com/bitcoin/bitcoin/pull/23511) | Yes |
```

You've dropped `N/A` from `Version Used` in the glibc column, which has broken the table.

![Screen Shot 2022-04-20 at 2 17 08 pm](https://user-images.githubusercontent.com/863730/164239050-bb520aaf-5bca-4a86-bbac-0ccd3b5945e8.png)

Reply:
Good catch! Fixed in abcb8769bf1d8ed227e6abddd7250a613e91277d